.github/workflows/build.yml

```diff
@@ -0,0 +1,32 @@
+name: showpcr-build
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "[0-9].[0-9]+.[0-9]+"
```

Comment:
I know I always use this format, but I know some people prefer v1.0.0.  Should we do that?  Support both?

Reply:
Happy to use v1.0.0